resources/graphClient/graphClient.css

```diff
@@ -12,12 +12,26 @@
 html {
     width: 100%;
```

Comment:
It's hard to read 'No data is loaded' text in certain themes:
![screen shot 2017-12-08 at 3 51 20 pm](https://user-images.githubusercontent.com/11282622/33789855-66c83348-dc30-11e7-847e-7ffdc99d825f.png)